### DISCUSSION TITLE
[HELP] Unable to access Student Developer Pack

### DISCUSSION THREAD

**Discussion by RupomChowdhury** [community/community#48669] [GitHub Education]
I am a verified student. Despite having a verified student account, I am unable to access the Developer Pack. The Pro Plan is active for me. However, something is wrong and I am unable to access the Developer Pack. 
Please help me asap!

Thank you

**Comment by RupomChowdhury** [comment]
![Screenshot (316)](https://user-images.githubusercontent.com/65197770/221444787-dd1b0220-6471-4e5f-9d91-4cf0f1cc573a.png)
![Screenshot (315)](https://user-images.githubusercontent.com/65197770/221444802-dade49f4-ce60-4e68-afa1-c024014e18e7.png)
![Screenshot (313)](https://user-images.githubusercontent.com/65197770/221444805-2eb20beb-90a0-4bb1-87bf-5946566972ce.png)
![Screenshot (314)](https://user-images.githubusercontent.com/65197770/221444808-223a87eb-e6a7-4c12-8f29-e949d54782a5.png)

**Comment by samus-aran** [comment]
Hi @RupomChowdhury  👋!

Once you become a verified student you have access to [Global Campus](https://education.github.com/globalcampus/student). 

GitHub Global Campus is a portal from which you can access your GitHub Education benefits and resources, all in one place. On the GitHub Global Campus portal, students can:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

It's great to have you in the GitHub Education community and we hope to see you around 👋 Let us know if any of these links don't work 👍

**Reply by RupomChowdhury** [reply]
Hello @samus-aran 

I am already verified by your system and also have PRO plan active. But still unable to access the developer pack or Global Campus. Once I go there it ask me to verify agin. But there no option to verify this existing verified account. I attached screenshots previously in this case.
What to do now? Please help!

Thanks.

**Reply by samus-aran** [reply]
Hi @RupomChowdhury 

Thanks for confirming this. In your screenshot it shows the latest attempt was "Rejected" can you provide more detail on that application?

**Reply by RupomChowdhury** [reply]
Hello @samus-aran 

Before that latest attempt(rejected) I was also unable to access the global campus/developer pack even after having student verified and PRO plan active that time. That's why I tried to re-verify but mistakenly I selected faculty instead of student option.

Thanks

![Request-a-discount-GitHub-Education](https://user-images.githubusercontent.com/65197770/222647155-64ca766b-5227-46bd-93dd-5c4936f42f95.png)
![RupomChowdhury-Rupom-](https://user-images.githubusercontent.com/65197770/222647185-43baddf0-8dbd-4899-bc27-c938c6b34c85.png)
![Billing](https://user-images.githubusercontent.com/65197770/222647239-b8599cf5-033e-4041-8f0c-b158fd6bb41c.png)

**Reply by RupomChowdhury** [reply]
@samus-aran where are you?

**Reply by RupomChowdhury** [reply]
Bug

**Reply by samus-aran** [reply]
Hi @RupomChowdhury 👋 

Did you close this discussion on purpose? If not then feel free to reopen it.

Regards to submitting a faculty application we'll investigate this and will provide an update here when we have one. 👍
